src/main/java/chipchat/action/CommandType.java

```diff
@@ -0,0 +1,20 @@
+package chipchat.action;
+
+import chipchat.exception.ChipchatException;
+
+public enum CommandType {
+    BYE, LIST, DELETE, MARK, UNMARK, TODO, DEADLINE, EVENT;
+
+    public static CommandType matchTaskType(String str) throws ChipchatException {
+        switch(str.toUpperCase()){
+            case "T":
```

Comment:
I THINK for our mod, we don't have indentation for switch and case hahaha. You can check the coding style for this mod again!